src/test/java/de/retest/recheck/RecheckImplTest.java

```diff
@@ -102,6 +108,58 @@ void calling_with_no_GM_should_produce_better_error_msg( @TempDir final Path roo
 
 	}
 
+	@Test
+	void no_golden_master_error_message_should_be_formatted_properly() throws Exception {
+		final ActionReplayResult actionReplayResult0 = mock( ActionReplayResult.class );
+		when( actionReplayResult0.getGoldenMasterPath() ).thenReturn( "/gm/path/0" );
+
+		final ActionReplayResult actionReplayResult1 = mock( ActionReplayResult.class );
+		when( actionReplayResult1.getGoldenMasterPath() ).thenReturn( "/gm/path/1" );
+
+		final TestReplayResult testReplayResult = new TestReplayResult( "test-name", 1 );
+		testReplayResult.addAction( actionReplayResult0 );
+		testReplayResult.addAction( actionReplayResult1 );
+
+		final RecheckImpl cut = new RecheckImpl();
+
+		final String errorMessage = cut.getNoGoldenMasterErrorMessage( testReplayResult );
+		assertThat( errorMessage ).isEqualTo( "'de.retest.recheck.RecheckImplTest':\n" //
+				+ NoGoldenMasterActionReplayResult.MSG_LONG + "\n" //
+				+ "/gm/path/0\n" //
+				+ "/gm/path/1" );
+	}
+
+	@Test
+	void inserted_and_deleted_differences_error_message_hould_be_formatted_properly() throws Exception {
+		final IdentifyingAttributes identifyingAttributes = mock( IdentifyingAttributes.class );
+		when( identifyingAttributes.getPath() ).thenReturn( "foo/bar/baz" );
+
+		final Element absent = null;
+		final Element present = mock( Element.class );
+		when( present.getIdentifyingAttributes() ).thenReturn( identifyingAttributes );
+
+		final InsertedDeletedElementDifference insertion =
+				InsertedDeletedElementDifference.differenceFor( absent, present );
+		final InsertedDeletedElementDifference deletion =
+				InsertedDeletedElementDifference.differenceFor( present, absent );
+		final LeafDifference ignore = mock( LeafDifference.class );
```

Comment:
Could you add a comment on why you added this `ignore`? Or maybe even make a separate test?